tests/ui/arch/vector_insert_dynamic.rs

```diff
@@ -0,0 +1,12 @@
+// Test `OpVectorInsertDynamic`
+// build-pass
+
+use spirv_std::arch;
+
+#[spirv(fragment)]
+pub fn main() {
+    let vector = glam::Vec2::new(1.0, 2.0);
+    let expected = glam::Vec2::new(1.0, 3.0);
+    let new_vector = unsafe { arch::vector_insert_dynamic(vector, 1, 3.0) };
+    assert!(new_vector == expected);
```

Comment:
This seems to be missing the `UVec` case from the original.
**EDIT**: ugh there are actually two "originals".